platform/source/minimal-printf/mbed_printf_implementation.c

```diff
@@ -194,17 +194,23 @@ static void mbed_minimal_formatted_string_unsigned(char *buffer, size_t length,
 /**
  * @brief      Print hexadecimal.
  *
- * @param      buffer  The buffer to store output (NULL for stdout).
- * @param[in]  length  The length of the buffer.
- * @param      result  The current output location.
- * @param[in]  value   The value to be printed.
- * @param      upper   Flag to print the hexadecimal in upper or lower case.
+ * @param      buffer           The buffer to store output (NULL for stdout).
+ * @param[in]  length           The length of the buffer.
+ * @param      result           The current output location.
+ * @param[in]  value            The value to be printed.
+ * @param      upper            Flag to print the hexadecimal in upper or lower case.
+ * @param      width_modifier   The width modifier. Only even values are supported. (0 equals 16, 16 max)
```

Comment:
Maybe the parameter description is a little bit confusing. If an odd number is given it is just rounded down to the next even integer since there is a divided by 2 in the function.
Sure I will rename it.

Reply:
I updated the parameter description.